packages/components/src/range-control/index.js

```diff
@@ -48,6 +48,7 @@ const BaseRangeControl = forwardRef(
 			allowReset = false,
 			beforeIcon,
 			className,
+			currentInput,
```

Comment:
I'm a bit confused. What is this prop? Why is it needed? How do we document it?

Reply:
To be honest... I'm not sure. But it was from the original `RangeControl` before the refactor 🙃 . I brought it back to preserve the value logic it had

Reply:
It's nor used or documented in Gutenberg so maybe we can pretend it was never there and forget about it?